recipes/tarball.rb

```diff
@@ -177,7 +177,7 @@
   only_if { ::File.readlines('/etc/pam.d/su').grep(/# #{pam_limits}/).any? }
 end
 
-if node['cassandra']['use_systemd'] == false
+if node['cassandra']['use_initd'] == true
```

Comment:
I've missed this in the original version, let's make this `node['cassandra']['use_initd']`.